.ci/scripts/test_wheel_package_qnn.sh

```diff
@@ -0,0 +1,203 @@
+        config=ExecutorchBackendConfig(extract_delegate_segments=False)
+    )
+    save_pte_program(executorch_program, "linear", args.output_folder)
+
+if __name__ == "__main__":
+    main()
+EOF
+
+# ----------------------------
+# Wheel build and .so checks
+# ----------------------------
+echo "=== Building Wheel Package ==="
+# pip install torch=="2.9.0.dev20250801" --index-url "https://download.pytorch.org/whl/nightly/cpu"
+source .ci/scripts/utils.sh
+install_executorch
+EXECUTORCH_BUILDING_WHEEL=1 python setup.py bdist_wheel
+unset EXECUTORCH_BUILDING_WHEEL
+
+WHEEL_FILE=$(ls dist/*.whl | head -n 1)
+echo "Found wheel: $WHEEL_FILE"
+
+# ----------------------------
+# Check wheel does NOT contain qualcomm/sdk
+# ----------------------------
+echo "Checking wheel does not contain qualcomm/sdk..."
+SDK_FILES=$(unzip -l "$WHEEL_FILE" | awk '{print $4}' | grep "executorch/backends/qualcomm/sdk" || true)
+if [ -n "$SDK_FILES" ]; then
+    echo "ERROR: Wheel package contains unexpected qualcomm/sdk files:"
+    echo "$SDK_FILES"
+    exit 1
+else
+    echo "OK: No qualcomm/sdk files found in wheel"
+fi
+
+# ----------------------------
+# Check .so files in the wheel
+# ----------------------------
+echo "Checking for .so files inside the wheel..."
+WHEEL_SO_FILES=$(unzip -l "$WHEEL_FILE" | awk '{print $4}' | grep "executorch/backends/qualcomm/python" || true)
+if [ -z "$WHEEL_SO_FILES" ]; then
+    echo "WARNING: No .so files found in wheel under executorch/backends/qualcomm/python"
+    exit 1
+else
+    echo "Wheel contains the following .so files:"
+    echo "$WHEEL_SO_FILES"
+fi
+
+# ----------------------------
+# Helpers
+# ----------------------------
+get_site_packages_dir () {
+  local PYBIN="$1"
+  "$PYBIN" - <<'PY'
+import sysconfig, sys
+print(sysconfig.get_paths().get("purelib") or sysconfig.get_paths().get("platlib"))
+PY
+}
+
+run_core_tests () {
+  local PYBIN="$1"      # path to python
+  local PIPBIN="$2"     # path to pip
+  local LABEL="$3"      # label to print (conda/venv)
+
+  echo "=== [$LABEL] Installing wheel & deps ==="
+  "$PIPBIN" install --upgrade pip
+  "$PIPBIN" install "$WHEEL_FILE"
+  "$PIPBIN" install torch=="2.9.0.dev20250801" --index-url "https://download.pytorch.org/whl/nightly/cpu"
+  "$PIPBIN" install --pre torchao --index-url "https://download.pytorch.org/whl/nightly/cpu"
+
+  echo "=== [$LABEL] Import smoke tests ==="
+  "$PYBIN" -c "import executorch; print('executorch imported successfully')"
+  "$PYBIN" -c "import executorch.backends.qualcomm; print('executorch.backends.qualcomm imported successfully')"
+
+  echo "=== [$LABEL] List installed executorch/backends/qualcomm/python ==="
+  local SITE_DIR
+  SITE_DIR="$(get_site_packages_dir "$PYBIN")"
+  local SO_DIR="$SITE_DIR/executorch/backends/qualcomm/python"
+  ls -l "$SO_DIR" || echo "Folder does not exist!"
+
+  echo "=== [$LABEL] Run export script to generate linear.pte ==="
+  (cd "$REPO_ROOT" && "$PYBIN" "$REPO_ROOT/script.py")
```

Comment:
nit: Maybe write this output to tmp? Just to avoid any name collison and polluting repo directory. Also consideing naming it script_qnn_wheel_test.py or something for name collision avoidance